Frontend/ui-customer/bower.json

```diff
@@ -24,15 +24,15 @@
     "purescript-strings": "^4.0.1",
     "purescript-enums": "^4.0.1",
     "purescript-foreign-generic": "^10.0.0",
-    "purescript-beckn-common": "git@ssh.bitbucket.juspay.net:bec/atlas-ui-common.git#main",
+    "purescript-beckn-common":"../ui-common",
     "purescript-tracker": "https://github.com/juspay/purescript-tracker.git#master"
   },
   "devDependencies": {
     "purescript-psci-support": "^4.0.0"
   },
   "resolutions": {
     "purescript-typelevel-prelude": ">=4.0.0 < 6.0.0",
-    "purescript-presto-dom": "nammaYatri",
+    "purescript-presto-dom": "main",
```

Comment:
is this tested? switching to main branch? did we replicated nammaYatri to main branch here?